tendermint/src/rpc/event_listener.rs

```diff
@@ -73,9 +73,23 @@ impl EventListener {
             .await
             .ok_or_else(|| RPCError::websocket_error("web socket closed"))??;
         match serde_json::from_str::<JsonRPCBlockResult>(&msg.to_string()) {
-            Ok(data) => Ok(Event::JsonRPCBlockResult(data)),
+            Ok(data) => {
+                if let Some(data) = data.0.result {
+                    return Ok(Event::JsonRPCBlockResult(data));
```

Comment:
It easier to see that this contains jsonrpcWrapper.result:
```suggestion
                if let Some(result) = data.0.result {
                    return Ok(Event::JsonRPCBlockResult(result));
```